observations/atmosphere/aircraft.yaml.j2

```diff
@@ -158,199 +564,412 @@
         options:
           xvar:
             name: MetaData/pressure
-          xvals: [70000, 65000, 60000, 55000, 50000, 45000, 40000, 35000, 30000, 25000,
-            20000, 15000, 10000, 7500, 5000]
-          errors: [2.4, 2.5, 2.6, 2.7, 2.8, 2.95, 3.1, 3.25, 3.4, 3.175, 2.95, 2.725,
-            2.5, 2.6, 2.7]
+          xvals: [110000, 70000, 65000, 60000, 55000, 50000, 45000, 40000, 35000, 30000, 25000, 20000, 15000, 10000, 7500, 5000]
+          errors: [2.4,   2.4,   2.5,   2.6,   2.7,   2.8,   2.95,  3.1,  3.25,   3.4,   3.175, 2.95,  2.725, 2.5,   2.6,  2.7]
     where:
     - variable:
         name: ObsType/windEastward
       is_in: 232
-    # Reject when difference of wind direction is more than 50 degrees.
-  - filter: Bounds Check
+
+  # Reject where QC is 9-15
+  - filter: Perform Action
+    filter variables:
+    - name: windEastward
+    - name: windNorthward
+    where:
+    - variable: PreQC/windNorthward
+      is_in: 9-15
+    action:
+      name: reject
+    defer to post: true
+
+#  # errormod
```

Comment:
Same question as temperature `errormod.`  Do we want to make GSI's `errormod ` for aircraft work in UFO?

Reply:
Per today's meeting, it's possible in the future we dive back into this.